src/locales/ru/messages.po

```diff
@@ -1417,7 +1417,8 @@ msgstr "Поскольку вы не установили срок финанс
 
 #: src/components/v1/FundingCycle/FundingCyclePreview.tsx
 msgid "Some funding cycle properties may indicate risk for project contributors."
-msgstr ""
+msgstr "Некоторые свойства цикла финансирования могут указывать на риск\n"
```

Comment:
@johnnyd-eth could you please remove this `\n` character from this Russian translation in crowdin?